tools/wptrunner/wptrunner/wptlogging.py

```diff
@@ -28,6 +29,22 @@ def setup_stdlib_logger():
     logging.root = stdadapter.std_logging_adapter(logging.root)
 
 
+def set_default_log_formatter(kwargs):
+    """When --verify is set and no default log formatter was set,
+    set a default log formatter to mach and its value is sys.stdout.
+    This is useful for ./wpt run --verify."""
+    log_formatters = ("raw", "unittest", "xunit", "html", "mach", "tbpl",
+                      "grouped", "errorsummary")
```

Comment:
[Also note `--log-wptreport` and `--log-wptscreenshot`.](https://github.com/web-platform-tests/wpt/blob/26c03768165078f56415db91c78ff64bcf9cc142/tools/wptrunner/wptrunner/wptcommandline.py#L329-L330) Maintained here, a list of formatters here is bound to go stale; do you think you could avoid it by inspecting the `commandline` module of the `mozlog` package? It's a singleton, so I'm not sure where would be the most appropriate place to import it.

Reply:
I didn't notice the `--log-wptreport` and `--log-wptscreenshot`. The `log_formatters` is from the `commandline` module in `mozlog` and this is the initial value. Even so, I realize it's not a good idea to hard-code here as it might be updated at other places. I'll avoid by importing `commandline` from `mozlog` package and using `commandline.log_formatters`.